### PR TITLE
feat: Batch mint tokens with CSV upload

### PR DESCRIPTION
**Pull request by lambdahands**

### CODE REVIEW

**Comment by obstropolos** [comment]
- Currently having issues using the csv format in the current testnet deployment
- Additionally, no metadata shows up for the asset's creator on the asset page when minting from CSV

Screen Shot 2021-05-17 at 10 56 01 PM

**Comment by paulo60pg** [comment]
Doesn't look to be working for now on florence testnet except if I'm doing something wrong.

File: src/components/CreateNonFungiblePage/FileUpload.tsx [hunk 1]
```diff
@@ -171,3 +185,88 @@ export default function FileUpload() {
...
+      );
+    },
+    [dispatch]
+  );
+
+  const { getRootProps, getInputProps } = useDropzone({
+    onDrop,
+    maxFiles: 1,
+    maxSize: 30 * 1024 * 1024,
+    accept: ['text/csv']
```
Comment (ricklove) [inline src/components/CreateNonFungiblePage/FileUpload.tsx:208]: On windows/chrome, the file type is unknown for a csv file, so this filter prevents it from working (I had to comment this out):

image

**Comment by ricklove** [comment]
> Doesn't look to be working for now on florence testnet except if I'm doing something wrong.

The rpc for florencenet seems to be down. (This wasn't introduced in this PR, but when moving from edonet to florencenet)

**Comment by ricklove** [comment]
I was able to mint the tokens in testnet after changing the csv to use the testnet contract address: See #431

**Comment by ricklove** [comment]
Random order of minted tokens (perhaps sort by token id descending)?

image

**Comment by ricklove** [comment]
Attributes are duplicated:
image

File: src/components/CreateNonFungiblePage/FileUpload.tsx (hunk 1 above)
Comment (ricklove) [reply src/components/CreateNonFungiblePage/FileUpload.tsx:208]: Fixed in #431 (disabled file filter)

**Review by obstropolos — APPROVED** [review]
Tested on both florencenet and mainnet implementations with batching - metadata fill-in and attributes all working.

**Pull request merged**